dockereve-master/.env

```diff
@@ -0,0 +1,5 @@
+MONGODB_HOST=mongodb
+MONGODB_PORT=27017
+
+API_TOKEN=afaketoken
+API_URL=http://example.com/url/to/docs
```

Comment:
Actually if you change this to `http://localhost/docs/api things should work out of the box in dev.